drift/views/admin.py

```diff
@@ -0,0 +1,24 @@
+from flask import Blueprint, jsonify
+
+from drift.version import app_version
+
+section = Blueprint("internal_admin", __name__)
+
+
+@section.before_app_request
+def status():
+    """Return"""
+    return jsonify(
+        {
+            "drift_version": app_version,
```

Comment:
can you indicate somewhere that this is test data?

Reply:
yeah, adding a comment to that effect is a good idea